lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -85,11 +94,13 @@ def get_upload_token(app_id, binary_path)
             request.headers["Authorization"] = "Bearer " + @auth_token
           end
         rescue Faraday::ResourceNotFound
-          UI.crash!("#{ErrorMessage::INVALID_APP_ID}: #{app_id}")
+          UI.user_error!("#{ErrorMessage::INVALID_APP_ID}: #{app_id}")
+        rescue Faraday::ForbiddenError
```

Comment:
Technically any call can raise a `Faraday::ForbiddenError` error, so we should either rescue it in every call or just let it fail loudly. I'm okay with letting it fail loudly i.e. removing the `rescue Faraday::ForbiddenError`.

Reply:
The reason I wanted to catch this error is because using the wrong service account credentials will allow the user to successfully authenticate but throw a 403 when trying to get the app token, so I wanted to make the error message clearer to reflect that.

Reply:
Gotcha. That makes sense. The reason I think I'm okay with not catching this and just failing loudly is because a generic 403 can be returned for a lot of different reasons, not just when a developer is using the wrong service account credentials. For example, if you use a refresh token for a user that doesn't have access to the app that will return a 403. Or if a developer is using the correct service account credentials but doesn't have the right permissions a 403 will be returned (right now this isn't possible i.e. anyone who has access can at least check the upload status, but in the future you can imagine a world where there is an "uploader role" that only allows uploading binaries and "manager" role which allows managing testers).